mlrun/lists.py

```diff
@@ -179,10 +179,14 @@ def to_rows(self):
             "producer": ["producer", "spec.producer"],
             "sources": ["sources", "spec.sources"],
             "labels": ["labels", "metadata.labels"],
+            "uri": ["uri", "uri"],
         }
         for artifact in self:
             fields_index = 0 if is_legacy_artifact(artifact) else 1
             row = [get_in(artifact, v[fields_index], "") for k, v in head.items()]
+            artifact_uri = dict_to_artifact(artifact).uri
+            last_index = len(row) - 1
+            row[last_index] = artifact_uri
```

Comment:
w.r.t @theSaarco I think it is also relebvant to mention why we need to resolve the location±uri here